apps/pie-docs/src/engineers/getting-started/overview.md

```diff
@@ -14,13 +14,13 @@ If you're just starting out with PIE, here's a list of recommended resources to
 
 ---
 
-## Code and Project Structure
+## PIE component systems
```

Comment:
Can we come up with a clearer title? This one doesn't give me many hints about what the content will be talking about

Reply:
Would it work to say `PIE component libraries` ?
Or if we include this content as part of the introduction, we can divided like so:
(If this does not work, let me know a suggestion :) )
<img width="810" alt="Screenshot 2024-07-17 at 11 59 32" src="https://github.com/user-attachments/assets/429ecaad-ba0d-430e-9529-7a2f124b1133">

Reply:
Decided to remove the smaller headings but left the new content as part of the Introduction section.